src/domain/networking/packets/EntityData.ts

```diff
@@ -523,44 +107,26 @@ type CommonEntityProperties = {
     certificateID: string | undefined;
     certificateType: string | undefined;
     staticCertificateVersion: number | undefined;
+    shapeType?: number | undefined;
+    compoundShapeURL?: string | undefined;
+    color?: color | undefined;
+    textures?: string | undefined;
+    modelURL?: string | undefined;
+    modelScale?: vec3 | undefined;
+    jointRotationsSet?: boolean[] | undefined;
+    jointRotations?: quat[] | undefined;
+    jointTranslationsSet?: boolean[] | undefined;
+    jointTranslations?: vec3[] | undefined;
+    groupCulled?: boolean | undefined;
+    relayParentJoints?: boolean | undefined;
+    blendShapeCoefficients?: string | undefined;
+    useOriginalPivot?: boolean | undefined;
+    animation?: AnimationProperties | undefined;
+    shape?: Shape | undefined;
+    alpha?: number | undefined;
```

Comment:
To avoid repetition, which will only get worse as further entity types are added, I think we could ...
- Remove the Model and Shape entity properties from here.
- Rename this type from `EntityDataDetails` to `EntityCommonData`.
- Add the following line:
```
type EntityDataDetails = EntityCommonData | ShapeEntityProperties | ModelEntityProperties;
```
- Perhaps also add the following line, to provide an alias (we want to use "EntityProperties" in the SDK JSDoc, iu think:
```
type EntityProperties = EntityDataDetails;
```

Reply:
The above suggestion should read:
```
type EntityDataDetails = EntityCommonData & ShapeEntityProperties & ModelEntityProperties;
```